### PR TITLE
Add more smoke tests

### PR DESCRIPTION
**Pull request by girazoki**
Adds three smoke tests regarding:
- issuance between blocks is correct
- proxies consistency
- treasury payment consistency

And fixes a few things from existing tests, specially those related to payment

### CODE REVIEW

**Review by tmpolaczyk — APPROVED** [review]

File: test/suites/smoke-test-common/test-configuration-consistency.ts [hunk 1]
```diff
@@ -58,8 +58,13 @@ describeSuite({
 
         it({
             id: "C03",
-            title: "Config all registered paras should be filled if more than min collators in orchestrator",
+            title: "Config registered paras should be filled if more than min collators in orchestrator",
             test: async function () {
+                const currentBlock = (await api.rpc.chain.getBlock()).block.header.number.toNumber();
+
+                const blockToCheck = Math.trunc(currentBlock / Number(blocksPerSession)) * Number(blocksPerSession);
```
Comment (fgamundi) [inline test/suites/smoke-test-common/test-configuration-consistency.ts:65]: Nice way to get the block

File: test/suites/smoke-test-common/test-configuration-consistency.ts [hunk 2]
```diff
@@ -69,10 +74,16 @@ describeSuite({
                     await api.query.authorityAssignment.collatorContainerChain(sessionIndex + 1)
                 ).toJSON();
 
-                // If we have container chain collators, is because we at least assigned min to orchestrator
-                if (
-                    Object.keys(authorities["orchestratorChain"]).length > config["minOrchestratorCollators"].toNumber()
-                ) {
+                const currentAuthorities = await api.query.session.validators();
+
+                const currentCollatorNumber = Math.min(currentAuthorities.length, config.maxCollators);
```
Comment (fgamundi) [inline test/suites/smoke-test-common/test-configuration-consistency.ts:79]: In what case can the authorities be greater than max collators?

File: test/suites/smoke-test-common/test-proxy.ts [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                if (count % (10 * limit) == 0) {
+                    log(`Retrieved ${count} proxies`);
+                }
+            }
+
+            // TEMPLATE: Adapt proxies
+            log(`Retrieved ${count} total proxies`);
+        }, 30_000);
+
+        // TEMPLATE: Give details about what you are testing
```
Comment (fgamundi) [inline test/suites/smoke-test-common/test-proxy.ts:64]:
```suggestion
```

**Review by fgamundi — APPROVED** [review]

File: test/suites/smoke-test-common/test-configuration-consistency.ts (hunk 2 above)
Comment (girazoki) [reply test/suites/smoke-test-common/test-configuration-consistency.ts:79]: probably in none, you are right

Comment (girazoki) [reply test/suites/smoke-test-common/test-configuration-consistency.ts:79]: maxCollators defines how many of currentAuthorities we take, where current authorities are invulnerables + staking collators. so we can have 100 potential candidatews, but maxCollators be only 70

Comment (fgamundi) [reply test/suites/smoke-test-common/test-configuration-consistency.ts:79]: I see, thanks!

**Pull request merged**